md/single-sign-on-with-cas.md

```diff
@@ -139,8 +136,8 @@ From 7.3.0, this configuration requires not to have initialized the platform or
 2. In the `CasLoginModule` configuration, check that the `principalGroupName` property is set to `CallerPrincipal`.  
    This is required to retrieve the username from the Bonita application.  
    Bonita BPM uses the CAS LoginModule in the JASIG implementation, so see the CAS LoginModule section of the [Jasig documentation](https://wiki.jasig.org/display/CASC/JAAS+Integration) for more information.
-3. Put the CAS client core jar (currently `cas-client-core-3.2.1.jar`) in the `/lib` folder.
-4. Put `commons-logging-1.1.jar` in the `/lib` folder.
+3. Copy `cas-client-core-3.3.1.jar` from `BonitaBPMSubscription-7.3.x-deploy` into the `/lib` directory.
```

Comment:
This doc is only valid up to 7.3. 
Starting from 7.4 there are other minor files to move and modify
So, I would leave it like that because it will remain part of the 7.3-doc only.
What do you think?